apps/meteor/app/api/server/api.ts

```diff
@@ -820,7 +821,7 @@ export class APIClass<TBasePath extends string = ''> extends Restivus {
 			},
 		);
 
-		const logout = function (this: Restivus): { status: string; data: { message: string } } {
+		const logout = async function (this: Restivus): Promise<{ status: string; data: { message: string } }> {
```

Comment:
Is the place using this func awaiting it?

Reply:
yes